src/pages/SummaryVotes.js

```diff
@@ -86,6 +89,12 @@ export const SummaryVotes = hh(class SummaryVotes extends Component {
     });
   }
 
+  getPowerBiDashboard() {
+    Config.getPowerBiUrl().then(url => {
+      this.setState({powerBiUrl: url})
```

Comment:
Codacy fix (https://app.codacy.com/gh/DataBiosphere/duos-ui/pullRequest?prid=6166400):
```suggestion
      this.setState({powerBiUrl: url});
```